kitchen.yml

```diff
@@ -68,6 +68,11 @@ platforms:
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
 
+  - name: amazonlinux-2
+    driver:
+      image: dokken/amazonlinux-2
+      pid_one_command: /usr/lib/systemd/systemd
```

Comment:
Minor nitpick, please but this near the top of the list so that this is alphabetized. Same with the GH actions yaml.

Reply:
Done! Note those lists are not alphabetized now (debian after centos) but I wasn't sure if I should address that.